sync_db.php

```diff
@@ -0,0 +1,49 @@
+<?php
+require 'lib/global.php';
+$pubmed=new PHPMed();
+$researchers=new NGIresearchers();
+$publications=new NGIpublications();
+
+if($USER->auth>0) {
+
+} else {
+	// Not logged in
+	header('Location:login.php');
+}
+
+
+// Render Page
+//=================================================================================================
+?>
+
+<!doctype html>
+<html class="no-js" lang="en" dir="ltr">
+
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="x-ua-compatible" content="ie=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title><?php echo $CONFIG['site']['name']; ?></title>
+	<link rel="stylesheet" href="css/foundation.css">
+	<link rel="stylesheet" href="css/app.css">
+	<link rel="stylesheet" href="css/icons/foundation-icons.css" />
+</head>
+
+<body>
+<?php require '_menu.php'; ?>
+
+<div class="row">
+	<br>
+	<div class="large-12 columns">
+		<span class="start_sync button">Begin syncing from publications.scilifelab.se</span>
+	</div>
+	<div class="card-section large-6 columns" id="sync_status_message"></div>
```

Comment:
This is why the status messages are off, the "card" should be a set of nested divs, see [docs](https://foundation.zurb.com/sites/docs/card.html)

Reply:
Ah, good to know! I just copy/pasted most of the css classes.  But yeah, I don't think I will be able to make the css pretty anyway. I expect @ewels to sweep in and change a bunch of that kind of stuff in no time.

Reply:
Possibly, depends how much I end up actually using this myself 😅

Reply:
Excuse me for being such a github noob, but I guess I could merge this to my repo, make changes and then that can be added back? Or how does one do this the best way?

Reply:
That's an excellent question. I think you are able to edit it directly in the web editor at least (a setting when I open my pull request). I only tested to edit the pull request locally once and it didn't work. But you can always pull my code, modify it and then open a new pull request to share it back to the upstream repo.

Reply:
Wait.. I see now you use the ["callout" class](https://foundation.zurb.com/sites/docs/callout.html) in the javascript. Just remove the "card-section" class and it should work fine. Maybe adjust to large-12 to cover the whole width.

Reply:
> Or how does one do this the best way?

Usually we just request that the original author makes the changes. Ideally with many small comments added on the relevant locations. If it really is easier to do the changes yourself then it is possible. the web editor is the easiest method as @alneberg says. Alternatively, you can pull from @alneberg's fork to your fork, make the changes, make a pull request against _his_ fork, merge it, and the changes will then appear on this pull request. Phew 😅 - make sense? You see why we normally just request the changes now..